__tests__/app.test.js

```diff
@@ -18,4 +20,32 @@ describe("GET /api", () => {
         expect(endpoints).toEqual(endpointsJson);
       });
   });
+
+
+});
+
+describe("GET /api/topics", () => {
+  test("200: Responds with an array of topic objects", () => {
+    return request(app)
+      .get("/api/topics")
+      .expect(200)
+      .then(({ body: { topics } }) => {
+        expect(Array.isArray(topics)).toBe(true);
+        topics.forEach((topic) => {
```

Comment:
If you're going to use a forEach on an array you must assert the array has some contents inside of it. An empty array will be skipped and the assertions inside will also be skipped.

Reply:
I've added another assertion to check the array has some contents in it.